src/sentry/static/sentry/app/components/group/inboxBadges/inboxReason.tsx

```diff
@@ -23,57 +23,83 @@ type Props = {
 function InboxReason({inbox, fontSize = 'sm'}: Props) {
   const {reason, reason_details, date_added: dateAdded} = inbox;
 
+  const getCountText = (count: number) =>
+    count > 1000 ? `More than ${Math.round(count / 1000)}k` : `${count}`;
```

Comment:
nit: would prefer to have the 1000 as a constant

Reply:
👍